src/api/subscriptions.js

```diff
@@ -115,6 +115,12 @@ export class Subscriptions {
    */
   subscribe(skuOrSubscriptionRequest) {}
 
+  /**
+   * Starts the Contributions flow.
+   * @param {!OffersRequest=} opt_options
+   */
+  showContributions(opt_options) {}
```

Comment:
Seems like the name is not very clear. We are not showing contributions here themselves, but rather contributions options?

Reply:
Removed for now.